### PR TITLE
Uploading JavaScript Assignment 1 & 2

### PR DESCRIPTION
**Pull request by shradha2205**

### CODE REVIEW

File: Shradha/JavaScript/Assignment1/ques-1.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                z = a ;
+                a = b ;
+                b = z ;   
+                document.getElementById("pid").innerHTML = "<b>Method 1: With Temporary Variable</b><br> After Swapping (Method 1) - Value of A = " + a + " and Value of B = " + b ;
+            }
+            function swap(){
+                var a , b ;
+                a = parseInt(document.getElementById("num1").value);
+                b = parseInt(document.getElementById("num2").value);
+                b = a - b ;
```
Comment (Snehal-113) [inline Shradha/JavaScript/Assignment1/ques-1.html:23]: a = a+b;
b = a-b;
a = a-b

File: Shradha/JavaScript/Assignment2/ques-1.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <p class="m-3 p-2" id="pid"></b></p>
+        <script type="text/javascript">
+            // second largest number
+            function secHigh() {
+                var a = document.getElementById('num1').value;
+                var b = document.getElementById('num2').value;
+                var c = document.getElementById('num3').value;
+                var d = document.getElementById('num4').value;
+                var e = document.getElementById('num5').value;
+                var arr = [a, b, c, d, e];
```
Comment (Snehal-113) [inline Shradha/JavaScript/Assignment2/ques-1.html:53]: try this program without array

File: Shradha/JavaScript/Assignment2/ques-2.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            function percent(){
+                var a, b, c, d, e, total, percentage ;
+                a = Number(document.getElementById("sub1").value) ;
+                b = Number(document.getElementById("sub2").value) ;
+                c = Number(document.getElementById("sub3").value) ;
+                d = Number(document.getElementById("sub4").value) ;
+                e = Number(document.getElementById("sub5").value) ;               
+                total = a + b + c + d + e;
+                document.getElementById("pid").innerText = " Total Marks secured is: " + total ;
+                percentage = (total/500) * 100 ;
```
Comment (Snehal-113) [inline Shradha/JavaScript/Assignment2/ques-2.html:58]: round off the percentage, the grade is not calculated correctly

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Shradha/JavaScript/Assignment2/ques-1.html (hunk 2 above)
Comment (shradha2205) [reply Shradha/JavaScript/Assignment2/ques-1.html:53]: ok mam

**Pull request merged**